src/templates/hl7v2/Resources/ADT_A01/Patient.hbs

```diff
@@ -1,151 +1,139 @@
-{
-    "fullUrl":"urn:uuid:{{ID}}",
-    "resource":{
-        "resourceType": "Patient",
-        "id":"{{ID}}",
-        "identifier":
-        [
-            {
-                {{>DataType/CX.hbs CX=PID-2}},
-            },
-            {{#each (getFieldRepeats PID-3)}}
```

Comment:
investigate: spec/inventory tells repeatable ? auto : handwritten + lookup

Reply:
yes, it's repeatable, here is the [PID-3 Spec](https://hl7-definition.caristix.com/v2/HL7v2.3/Fields/PID.3)